make/DynamoRIOConfig.cmake.in

```diff
@@ -719,7 +725,7 @@ function (_DR_set_compile_flags target is_client extra_flags)
         endif()
       endif (NOT DEFINED DynamoRIO_VISATT)
       if (DynamoRIO_VISATT)
-        set(extra_flags "${extra_flags} -DUSE_VISIBILITY_ATTRIBUTES")
+        set(extra_defines ${extra_defines} USE_VISIBILITY_ATTRIBUTES)
```

Comment:
Consistency (not sure if it affects correctness too): value is put in a string at other spots, unlike here.

Reply:
Done.  It does not affect correctness.